src/components/Header.js

```diff
@@ -39,8 +41,26 @@ export default function(store){
         if (/^\d{5}$/.test(newZipcode) || !newZipcode) {
           store.dispatch('setFilters',{zipcode: newZipcode });
         }
+      },
+      headerHeight: function() {
```

Comment:
I'm not in love with this, but the header changes height several times as different assets come in, and setting it only once during `mount` isn't gonna be good enough. I'm open to better ideas though!